pacts/sirius-workflow-sirius.json

```diff
@@ -0,0 +1,636 @@
+{
```

Comment:
Is there a reason why this file is back? Github is telling me it is an addition.

Reply:
Oh no, I thought i'd deleted it!